### PR TITLE
Rel 467215 remove internal domain

### PR DESCRIPTION
**Pull request by aarongilbert123**

### CODE REVIEW

**Review by jake-martin — DISMISSED** [review]

**Review by TommyBigNose — APPROVED** [review]

File: Chef/Cookbooks/BaseImage/attributes/default.rb [hunk 1]
```diff
@@ -6,29 +6,30 @@
 default['windows']['user']['admin']['password'] = ENV['devvm_default_password']
 
 default['file']['installers']['default_destination_folder'] = 'C:\\Chef_Install'
+default['file']['installers']['devvm_installer_files_path'] = ENV['devvm_installer_files_path']
```
Comment (calimetikcura) [inline Chef/Cookbooks/BaseImage/attributes/default.rb:9]: Can we add a comment at the end of the line saying this env variable is stored on the DevVM Base Image.

File: Chef/Cookbooks/BaseImage/attributes/default.rb [hunk 2]
```diff
@@ -6,29 +6,30 @@
 default['windows']['user']['admin']['password'] = ENV['devvm_default_password']
```
Comment (calimetikcura) [inline Chef/Cookbooks/BaseImage/attributes/default.rb:6]: Can we add a comment at the end of the line saying this env variable is stored on the DevVM Base Image.

File: Chef/Cookbooks/Relativity/attributes/default.rb [hunk 3]
```diff
@@ -6,29 +6,30 @@
 default['windows']['user']['admin']['password'] = ENV['devvm_default_password']
```
Comment (calimetikcura) [inline Chef/Cookbooks/Relativity/attributes/default.rb:6]: Can we add a comment at the end of the line saying this env variable is stored on the DevVM Base Image.

File: Chef/Cookbooks/Relativity/attributes/default.rb [hunk 4]
```diff
@@ -6,29 +6,30 @@
 default['windows']['user']['admin']['password'] = ENV['devvm_default_password']
 
 default['file']['installers']['default_destination_folder'] = 'C:\\Chef_Install'
+default['file']['installers']['devvm_installer_files_path'] = ENV['devvm_installer_files_path']
```
Comment (calimetikcura) [inline Chef/Cookbooks/Relativity/attributes/default.rb:9]: Can we add a comment at the end of the line saying this env variable is stored on the DevVM Base Image.

**Review by calimetikcura — CHANGES_REQUESTED** [review]

File: Chef/Cookbooks/BaseImage/attributes/default.rb (hunk 1 above)
Comment (aarongilbert123) [reply Chef/Cookbooks/BaseImage/attributes/default.rb]: Done

File: Chef/Cookbooks/BaseImage/attributes/default.rb (hunk 2 above)
Comment (aarongilbert123) [reply Chef/Cookbooks/BaseImage/attributes/default.rb]: Done

File: Chef/Cookbooks/Relativity/attributes/default.rb (hunk 3 above)
Comment (aarongilbert123) [reply Chef/Cookbooks/Relativity/attributes/default.rb]: Done

File: Chef/Cookbooks/Relativity/attributes/default.rb (hunk 4 above)
Comment (aarongilbert123) [reply Chef/Cookbooks/Relativity/attributes/default.rb]: Done

**Review by calimetikcura — APPROVED** [review]

**Review by TommyBigNose — APPROVED** [review]

**Pull request merged**